Author: Roland Rosenfeld <roland@debian.org>
Description: Get rid of several "implicit declaration of function" warnings.

--- a/fig2dev/fig2dev.c
+++ b/fig2dev/fig2dev.c
@@ -42,7 +42,6 @@
 
 void	help_msg();
 void	depth_option();
-int	depth_filter(int);
 
 /* hex names for Fig colors */
 char	*Fig_color_names[] = {
--- a/fig2dev/fig2dev.h
+++ b/fig2dev/fig2dev.h
@@ -116,6 +116,7 @@
 extern void	unpsfont();
 extern void	print_comments();
 extern int	lookup_X_color();
+extern int	depth_filter(int);
 
 extern char	Err_badarg[];
 extern char	Err_mem[];
--- a/fig2dev/bound.c
+++ b/fig2dev/bound.c
@@ -17,6 +17,7 @@
 #include "fig2dev.h"
 #include "object.h"
 #include "bound.h"
+#include "localmath.h"
 
 extern int adjust_boundingbox;
 
--- a/fig2dev/dev/gendxf.c
+++ b/fig2dev/dev/gendxf.c
@@ -40,6 +40,7 @@
 
 #include "fig2dev.h"
 #include "object.h"
+#include "localmath.h"
 
 static void set_style();
 
--- a/fig2dev/dev/genpictex.c
+++ b/fig2dev/dev/genpictex.c
@@ -27,6 +27,7 @@
 #include "genlatex.h"
 #include "setfigfont.h"
 #include "texfonts.h"
+#include "localmath.h"
 
 #define UNIT "cm"       /* dip */
 #define CONVUNIT 2.54   /* dip */
--- a/fig2dev/dev/genpstex.c
+++ b/fig2dev/dev/genpstex.c
@@ -43,6 +43,7 @@
 #include "genpdf.h"
 #include "object.h"
 #include "texfonts.h"
+#include "setfigfont.h"
 
 extern double rad2deg;
 
--- a/fig2dev/localmath.h
+++ b/fig2dev/localmath.h
@@ -15,4 +15,5 @@
  *
  */
 
-void arc_tangent();
+void arc_tangent(double, double, double, double, double *, double *, int);
+void arc_tangent_int(double, double, double, double, int *, int *, int);
--- a/fig2dev/dev/gentextyl.c
+++ b/fig2dev/dev/gentextyl.c
@@ -28,6 +28,7 @@
 #include "fig2dev.h"
 #include "object.h"
 #include "texfonts.h"
+#include "localmath.h"
 
 static void putline();
 
--- a/transfig/sys.c
+++ b/transfig/sys.c
@@ -17,6 +17,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
 #include "transfig.h"
 
 #define MAXSYS 10000
--- a/transfig/transfig.c
+++ b/transfig/transfig.c
@@ -26,6 +26,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include "patchlevel.h"
 #include "transfig.h"
 
--- a/transfig/transfig.h
+++ b/transfig/transfig.h
@@ -50,8 +50,12 @@
 extern char *iname[];
 
 extern char *sysls(), *mksuff();
+extern void sysmv(char *);
+
 extern argument *arglist;
 extern char *txfile, *mkfile;
+extern void texfile(FILE *, char *, argument *);
+extern void makefile(FILE *, int, argument *);
 
 extern char *optarg;
 extern int optind;
